src/const/blob-types.const.js

```diff
@@ -0,0 +1,10 @@
+export const BLOB_TYPES = Object.freeze({
+  assetDescription: 'asset_description',
+  fundOverview: 'fund_overview',
+  fundUpdate: 'fund_update',
+  navUpdate: 'nav_update',
```

Comment:
we don't need `navUpdate` and `fundUpdate` types

Reply:
Removed